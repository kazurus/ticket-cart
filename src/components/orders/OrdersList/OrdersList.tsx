import React from 'react';

const OrdersList: React.FC = ({ children }) => (
  <div>
    <h3>Orders</h3>
    <div>{children}</div>
  </div>
);

export default OrdersList;
