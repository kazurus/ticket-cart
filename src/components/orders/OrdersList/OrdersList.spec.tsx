import React from 'react';
import { render } from '@testing-library/react';
import OrdersList from './OrdersList';

describe('OrdersList component', () => {
  test('should render props', () => {
    const { container } = render(
      <OrdersList>
        <p>Order children</p>
      </OrdersList>
    );

    expect(container).toMatchSnapshot();
  });
});
