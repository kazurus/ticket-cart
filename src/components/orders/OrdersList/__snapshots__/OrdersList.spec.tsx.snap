// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrdersList component should render props 1`] = `
<div>
  <div>
    <h3>
      Orders
    </h3>
    <div>
      <p>
        Order children
      </p>
    </div>
  </div>
</div>
`;
